### PR TITLE
feat(ingress): custom annotations for alb ingress

### PR DESCRIPTION
**Pull request by Hunter-Thompson**
Adds an example for the AWS ALB Ingress. Also adds annotations for the AWS ALB ingress.

```release-note
Custom annotations for AWS ALB Ingress
```

### CODE REVIEW

File: docs/examples/mattermost_full_aws_alb.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+metadata:
+  name: mm-example-full-alb                       # Name of the Mattermost cluster as shown in Kubernetes.
+spec:
+  image: mattermost/mattermost-enterprise-edition # Docker image for the app servers.
+  imagePullPolicy: IfNotPresent                   # Image Pull policy used by Mattermost deployment.
+  version: 5.28.0                                 # Docker tag for the image.
+  size: 5000users                                 # Size of the Mattermost installation, typically based on the number of users. This a is write-only field - its value is erased after setting appropriate values of resources. Automatically sets the replica and resource limits for Minio, databases and app servers based on the number provided here. Accepts 100users, 1000users, 5000users, 10000users, or 25000users. Manually setting replicas or resources will override the values set by 'size'.
+  awsLoadBalancerController:
+    enabled: true                                 # If disabled Ingress will not be created by Mattermost Operator.
+    host: example.mattermost-example.com          # Hostnames used for Ingress, e.g. example.mattermost-example.com. Required when Ingress is enabled.
```
Comment (mirshahriar) [inline docs/examples/mattermost_full_aws_alb.yaml:16]: This field is not available under `awsLoadBalancerController `

File: docs/examples/mattermost_full_aws_alb.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    hosts: 
+      - hostName: 'example.mattermost-example.com'
+    annotations: {}                               # Custom annotations propagated to Ingress resource.
```
Comment (mirshahriar) [inline docs/examples/mattermost_full_aws_alb.yaml:19]: This field is not available under `awsLoadBalancerController`

**Comment by Hunter-Thompson** [comment]
@mirshahriar I added a new field for annotations, and removed the `host` key.

**Review by gabrieljackson — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**